src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -665,6 +666,70 @@ struct jit_uni_def_conv_kernel_f32 : public jit_uni_def_conv_kernel, public jit_
     }
 };
 
+
+namespace {
+
+struct DefConvKey {
+    DnnlMemoryDescCPtr inp0;  // data
```

Comment:
shouldn't we also include into hash `group` value?

Reply:
No. It is defined by shape